### PR TITLE
add changes for demo with child input

### PR DESCRIPTION
**Pull request by trevorhuis-smoothstack**

### CODE REVIEW

File: src/app/agent/agent-dashboard/create-booking/create-booking.component.css [hunk 1]
```diff
@@ -102,4 +102,8 @@
 .custom-switch:hover .tooltiptext {
   visibility: visible;
   opacity: 1;
+}
+
+.ad-image {
+  background-image: url("../../../../assets/images/ad/miami_ad.JPG");
 }
```
Comment (justin-obrien-smoothstack) [inline src/app/agent/agent-dashboard/create-booking/create-booking.component.css:109]: What's this for?

File: src/app/agent/agent-dashboard/agent-dashboard.component.ts [hunk 2]
```diff
@@ -53,29 +60,24 @@ export class AgentDashboardComponent implements OnInit {
...
-
-  ngOnDestroy() {
-    document.getElementById("nav-agent").classList.remove("active");
-  }
+  } 
 
   adjustForMobile(width) {
     if(width < 992) {
       this.mobile = true;
-      this.moveSidebarToNav();
```
Comment (justin-obrien-smoothstack) [inline src/app/agent/agent-dashboard/agent-dashboard.component.ts:69]: Why is this no longer needed?

Comment (trevorhuis-smoothstack) [reply src/app/agent/agent-dashboard/agent-dashboard.component.ts:69]: It wasn't needed long ago, just forgot to remove it.

File: src/app/agent/agent-dashboard/create-booking/create-booking.component.css (hunk 1 above)
Comment (trevorhuis-smoothstack) [reply src/app/agent/agent-dashboard/create-booking/create-booking.component.css:109]: Good catch, it was for adding the image in a different way but it is no longer being used.

**Pull request merged**